Signalling/src/PlayerConnection.ts

```diff
@@ -210,7 +214,9 @@ export class PlayerConnection implements IPlayer, LogUtils.IMessageLogger {
 
     private onListStreamers(_message: Messages.listStreamers): void {
         const listMessage = MessageHelpers.createMessage(Messages.streamerList, {
-            ids: this.server.streamerRegistry.streamers.map((streamer) => streamer.streamerId)
+            ids: this.server.streamerRegistry.streamers
+                .filter((streamer) => streamer.streaming)
```

Comment:
How does a streamer become `streaming` again? Just sanity checking this.

Reply:
When the streamer responds to the endpoint id request its marked as streaming.

Reply:
Okay, I think that works well, we just will ensure we don't backport this one.